src/js/sdk.tsx

```diff
@@ -38,6 +39,7 @@ import { NATIVE } from './wrapper';
 const IGNORED_DEFAULT_INTEGRATIONS = [
   'GlobalHandlers', // We will use the react-native internal handlers
   'TryCatch', // We don't need this
+  'LinkedErrors', // We replace this with `NativeLinkedError`
```

Comment:
What's about https://github.com/getsentry/sentry-react-native/issues/2118
Does this PR help with that?